libevmasm/Assembly.cpp

```diff
@@ -574,6 +576,23 @@ LinkerObject const& Assembly::assemble() const
 			ret.bytecode.resize(ret.bytecode.size() + bytesPerTag);
 			break;
 		}
+		case JumpTablePushTag:
+		{
+			unsigned pushSize = (unsigned) i.jumpTableTags().size() * bytesPerJumpTableTag;
+			if (pushSize == 0)
```

Comment:
Can this happen? Should we maybe rather assert that there's at least one tag already when constructing JumpTablePushTag items?

Reply:
I tried to write this in a way that was as permissive as possible, so technically a jump table with no cases would work here. But since there is not much use for a zero-case jump table, I have added asserts when constructing a JumpTablePushTag and when doing the final assembly.